communication/socket/tcp/send/wsaioctl_transmitpackets_callbackfunc.yml

```diff
@@ -0,0 +1,28 @@
+rule:
+  meta:
+    name: obtains TransmitPackets callback function via WSAIoctl  
+    description: The TransmitPackets function transmits in-memory data or file data over a connected socket. The TransmitPackets function uses the operating system cache manager to retrieve file data, locking memory for the minimum time required to transmit and resulting in efficient, high-performance transmission.
+    namespace: communication/socket/tcp/send
+    author: jonathan.lepore@mandiant.com
+    scope: function 
+    references: https://docs.microsoft.com/en-us/windows/win32/api/mswsock/nc-mswsock-lpfn_transmitpackets
+    examples:
+      - 138C71E94961FE9E31CEC5DFBA62A639:0x100FF55
```

Comment:
please open a second PR against https://github.com/fireeye/capa-testfiles/ that adds this file